src/commands/mod.rs

```diff
@@ -736,6 +774,370 @@ impl DaemonControl {
             .map_err(CommandError::TxBroadcast)
     }
 
+    // This is like `create_spend` with slightly different parameters.
+    fn create_rbf_spend(
```

Comment:
For now, this is almost the same as `create_spend` with some slightly different parameters. Perhaps a follow-up PR could refactor to reduce duplicate code.

Reply:
After https://github.com/wizardsardine/liana/pull/821 with the addition of a change parameter, the only real difference between this method and `create_spend` will be:
- a `min_fee` parameter (I think we can avoid this parameter by just calling this function with an increasing feerate until it has the requried fee, which I'm already doing anyway given that the min fee depends on the size of the replacement transaction itself, see https://github.com/wizardsardine/liana/pull/816/files#diff-bf2f650cdc66abd13076c47b25e8c9f783f06ec335bbd30822fdd690547615fdR1100)
- whether to use a mix of mandatory and optional candidates or just those in `coins_outpoints` -> we could change the logic of `create_spend` so that optional candidates are used not when `coins_outpoints` is empty but when this extra Boolean parameter is `true`.

So I think by adding a Boolean parameter to `create_spend` in addition to #821, we could remove `create_rbf_spend`.

Reply:
> a min_fee parameter (I think we can avoid this parameter by just calling this function with an increasing feerate until it has the requried fee

I don't think we should do that.

But we can still have a `create_spend_internal` function which takes more parameter and is called by both `create_spend` and `rbf_psbt`.

Reply:
Looks good to me!

ICYMI: https://github.com/wizardsardine/liana/pull/816/files#diff-bf2f650cdc66abd13076c47b25e8c9f783f06ec335bbd30822fdd690547615fdR465-R466 the comment here is now outdated.

Reply:
Great, thanks for the quick feedback. I'll update that comment and check the others.